core/src/main/scala/com/yahoo/maha/core/query/druid/DruidQueryGenerator.scala

```diff
@@ -1100,6 +1151,11 @@ class DruidQueryGenerator(queryOptimizer: DruidQueryOptimizer
                               renderColumnWithAlias(fact, sourceCol, name, forPostAggregator = true)
                             }
                         }
+                      case DruidHyperUniqueTimeBucketedRollup(_) => {
```

Comment:
Do you need this here if you are just doing what the default case is doing?

Reply:
Removed it